### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by svitalis123**
# Vet clinic database: query and update animals table
## I accomplished

- [ ] Updated the animal's table by setting the species column to Digimon for all animals that have a name ending on mon.
- [ ] Updated the animal's table by setting the species column to pokemon for all animals that don't have species already set.
- [ ] Deleted all animals born after Jan 1st, 2022.
- [ ] Created a savepoint for the transaction.
- [ ] Updated all animals' weight to be their weight multiplied by -1.
- [ ] Rollbacked to the savepoint
- [ ] Updated all animals' weights that are negative to be their weight multiplied by -1.
- [ ] Committed transaction.
- [ ] Queried How many animals are there?
- [ ] Queried How many animals have never tried to escape?
- [ ] Queried What is the average weight of animals?
- [ ] Queried Who escapes the most, neutered or not neutered animals?
- [ ] Queried What is the minimum and maximum weight of each type of animal?
- [ ] Queried What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,25 @@
  SELECT * FROM animals WHERE neutered = True;
  SELECT * FROM animals WHERE NOT name =  'Gabumon';
  SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+ BEGIN;
+ UPDATE animals SET species = 'unspecified';
+ ROLLBACK;
```
Comment (Lucas-Masaba) [inline queries.sql:13]: - [ ] Just <strong>above</strong> this `ROLLBACK` on line 13, please verify that the changes were made by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.
- [ ] Just <strong>below</strong> this `ROLLBACK` on line 13, please verify that the changes were made by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,25 @@
...
+ BEGIN;
+ UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+ UPDATE animals SET species = 'pokemon' WHERE species = '';
+ COMMIT;
```
Comment (Lucas-Masaba) [inline queries.sql:17]: - [ ] Just <strong>above</strong> this `COMMIT` on line 17, please verify that the changes were made by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.
- [ ] Just <strong>below</strong> this `COMMIT` on line 17, please verify that the changes persisted by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @svitalis123  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost-there

 ## Highlights 🍾 
- All requirements were met. All you have to do is add the verifications as requested 🏄 
- Descriptive PR 📜 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi vitalis, 
## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**